parquet-benchmarks/src/main/java/org/apache/parquet/benchmarks/DataGenerator.java

```diff
@@ -126,14 +160,36 @@ public void cleanup()
   }
 
   public static void main(String[] args) {
-    DataGenerator generator = new DataGenerator();
+    DataGenerator generator = new DataGenerator(new Configuration());
+
     if (args.length < 1) {
-      System.err.println("Please specify a command (generate|cleanup).");
+      System.err.println("Please specify a command (generate VERSION|cleanup).");
       System.exit(1);
     }
 
     String command = args[0];
     if (command.equalsIgnoreCase("generate")) {
+      if (args.length < 2 || args.length > 3) {
+        System.err.println("Usage: generate VERSION [-randomData]\n");
+        System.err.println("Options:");
+        System.err.println("  VERSION         Use a specific parquet file version to generate data.");
+        System.err.println("                  - v1 for PARQUET_1_0 version.");
+        System.err.println("                  - v2 for PARQUET_2_0 version.");
+        System.err.println("  -randomData     This flag specifies if random data will be used to generate data.");
```

Comment:
Is `randomData` optional? When I only specify the version to generate some data (with a command that looks like ```.... org.apache.parquet.benchmarks.DataGenerator generate v2```) I get an exception

```Java
Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 2
	at org.apache.parquet.benchmarks.DataGenerator.main(DataGenerator.java:188)
```